Sources/Components/SaveSearchView/SaveSearchPromptView.swift

```diff
@@ -0,0 +1,100 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import UIKit
+
+// MARK: - SaveSearchPromptViewDelegate
+
+public protocol SaveSearchPromptViewDelegate: AnyObject {
+    func saveSearchPromptView(_ saveSearchPromptView: SaveSearchPromptView, didAcceptSaveSearch: Bool)
+}
+
+// MARK: - SaveSearchPromptView
+
+public class SaveSearchPromptView: UIView {
+
+    // MARK: - Public properties
+
+    public weak var delegate: SaveSearchPromptViewDelegate?
+
+    // MARK: - Private properties
+
+    private lazy var titleLabel: UILabel = {
+        let label = UILabel()
+        label.numberOfLines = 2
```

Comment:
This labels could use more than two lines if we use large fonts. It is a good idea to test how the view looks when turning on large fonts